lib/shopify_api/resources/ping/message.rb

```diff
@@ -1,9 +1,23 @@
 # frozen_string_literal: true
 
+require 'byebug'
+
 module ShopifyAPI
   module Ping
     class Message < Base
       self.prefix = "/admin/api/ping-api/v1/conversations/:conversation_id/"
+
+      def successful_delivery(options)
+        delivery_details = ShopifyAPI::Ping::DeliveryConfirmation.new(options)
+        delivery_details.save
```

Comment:
Does `delivery_details.save` not return a boolean already?

Reply:
Looks like it does, just updated it accordingly.